src/components/print/PrintDirective.js

```diff
@@ -541,6 +541,19 @@ goog.require('ga_urlutils_service');
         if (!layer.visible || layer.opacity == 0) {
           return;
         }
+        // Only print layer which have an extent intersecting the print extent
+        var layerExtent = layer.getExtent();
+        if (layerExtent) {
+          var intersection = ol.extent.getIntersection(layerExtent,
```

Comment:
No `ol.extent.intersects(extent1, extent2)`works only if `extents2` is completely enclosed in `extent1`, which is a bit too restrictive. If only a part of `extent2` is enclosed, I want it to be printed.